app/src/index.tsx

```diff
@@ -5,13 +5,19 @@ import App from './components/App';
 import * as serviceWorker from './services/serviceWorkers/serviceWorker';
 import Home from './scenes/Home';
 import { SingleRoute } from './interfaces';
+import Items from './scenes/Items';
 
 const routes: SingleRoute[] = [
   {
     path: '/academix',
     exact: true,
     component: Home,
   },
+  {
+    path: '/academix/:categoryId/:categoryName/:subCategoryId/:subCategoryName',
```

Comment:
Now I think we can replace it with something like this:
/academix/sub/:subCategoryId/:subCategoryName'

Because :categoryId an categoryName will be irrelevant when it comes to this view.

WDYT?

Reply:
yeah that’s perfect